addons/stock_account/models/account_move.py

```diff
@@ -93,7 +93,7 @@ def _stock_account_prepare_anglo_saxon_out_lines_vals(self):
         '''
         lines_vals_list = []
         for move in self:
-            if not move.is_sale_document(include_receipts=True) or not move.company_id.anglo_saxon_accounting:
+            if (not move.is_sale_document(include_receipts=True) or not move.company_id.anglo_saxon_accounting) and not already_out:
```

Comment:
seems weird to bypass this logic when already out, i think we should add more accounting entries if the product is already sold but. can you explain?